examples/suite_FV3_test.xml

```diff
@@ -10,7 +10,7 @@
   <group name="physics">
     <subcycle loop="1">
       <scheme>cnvc90</scheme>
-      <scheme>mp_thompson_hrrr</scheme>
```

Comment:
The standard suite should not contain any non-standard physics. I made the same mistake with the Thompson physics, when I discovered it I created a separate suite_FV3_test_thompson.xml. Your branch is not up to date, hence you don't see this. We need to create a suite_FV3_test_gfdl.xml file and leave suite_FV3_test.xml untouched.